Remove summary section on home page and redesign navbar

yb-andrew opened the pull request:
We wanted to put the 'Get Started' links above the fold and simplify the navbar links to link to Slack and Github.

_EDIT: Removed old screenshots_

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/YugaByte/yugabyte-db?pullRequest=1751) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/YugaByte/yugabyte-db?pullRequest=1751) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/YugaByte/yugabyte-db?pullRequest=1751) it.</sub>

yb-andrew commented:
Simplified navbar and added social media links. Slim down the homepage to bring the user guides above the fold.
Wide: 1080px
![docs-new-homepage](https://user-images.githubusercontent.com/50115930/61009221-db9b8d00-a326-11e9-9002-8841e3dfeca6.png)

Ultra-wide: 1920px
![docs-new-homepage-ultra](https://user-images.githubusercontent.com/50115930/61009285-10a7df80-a327-11e9-919e-e591ed087cf5.png)
